sql/hive/pom.xml

```diff
@@ -133,7 +133,6 @@
                 </goals>
                 <configuration>
                   <sources>
-                    <source>src/test/scala</source>
```

Comment:
It is not needed anymore. It did hurt you in the past because the add-sources was done by build-helper-maven-plugin only. Overriding the configuration here has overriden the setting from parent. That is why one had to repeat it all the time or the dir set in parent was lost.

Now however the addition happens because there are two plugins "adding sources".

BTW: We can keep it there. It does not hurt. It is just a cleanup.